core/validatorapi/router.go

```diff
@@ -367,7 +367,7 @@ func proposeBlock(p eth2client.BeaconBlockProposalProvider) handlerFunc {
 			}
 
 			return proposeBlockResponsePhase0{
-				Version: block.Version,
+				Version: "PHASE0",
```

Comment:
@corverroos i didn't added marshalling thing because we do marshalling of response as part of writeResponse.

Also didn't added any specific function to enforce this constant because anyways these fields will be exported so why having a seperate function for that when we can do that directly.

Reply:
fair enough